packages/cypress/internal/cypress-install.js

```diff
@@ -0,0 +1,163 @@
+const {spawnSync, spawn} = require('child_process');
```

Comment:
add a comment about when does this code run? I guess it's part of the repository rule, since it writes .bzl files?

Reply:
done